docs/src/pages/components/pickers/MaterialUIPickers.js

```diff
@@ -19,43 +20,42 @@ export default function MaterialUIPickers() {
   };
 
   return (
-    <MuiPickersUtilsProvider utils={DateFnsUtils}>
+    <MuiPickersLocalizationProvider dateAdapter={DateFnsAdapter}>
```

Comment:
Why rename the component?

```suggestion
    <LocalizationProvider dateAdapter={DateFnsAdapter}>
```

Reply:
Because it's concerned with localization now not utils.